### PR TITLE
last search tweaks 

### PR DESCRIPTION
**Pull request by skylarbarrera**
this should work g000d now

### CODE REVIEW

File: src/components/discover-sheet/DiscoverSearch.js [hunk 1]
```diff
@@ -126,7 +127,7 @@ export default function DiscoverSearch() {
   }, [searchQuery, setIsSearching, startQueryDebounce, stopQueryDebounce]);
 
   useEffect(() => {
-    if (!uniswapCurrencyListLoading && !isFetchingEns) {
+    if (!uniswapCurrencyListLoading || !isFetchingEns) {
```
Comment (jinchung) [inline src/components/discover-sheet/DiscoverSearch.js:130]: are we sure about this

**Comment by osdnk** [comment]
Can you explain this?

File: src/components/exchange/CurrencySelectionList.js [hunk 2]
```diff
@@ -64,10 +65,8 @@ const CurrencySelectionList = (
     >
       {showList && !showSkeleton && (
         <Centered flex={1}>
-          {showNoResults ? (
-            loading ? null : (
-              <NoCurrencyResults />
-            )
+          {showGhost ? (
```
Comment (jinchung) [inline src/components/exchange/CurrencySelectionList.js:68]: renamed "showNoResults" to "showGhost" bc it was a bit confusing bc the ghost is shown not just with no results showing

**Review by jinchung — APPROVED** [review]
🌮

**Pull request merged**